pkg/engine/mutate/strategicMergePatch.go

```diff
@@ -120,9 +119,14 @@ func strategicMergePatch(base, overlay string) ([]byte, error) {
 	return baseObj.Bytes(), err
 }
 
-func preProcessStrategicMergePatch(pattern, resource string) (*yaml.RNode, error) {
+func preProcessStrategicMergePatch(logger logr.Logger, pattern, resource string) (*yaml.RNode, error) {
 	patternNode := yaml.MustParse(pattern)
 	resourceNode := yaml.MustParse(resource)
-	err := preProcessPattern(patternNode, resourceNode)
+	err := preProcessPattern(logger, patternNode, resourceNode)
+	fmt.Println("1. --------------- ")
```

Comment:
@kacejot we should remove this log message

Reply:
Sure